src/modules/connections/models/did/DidDoc.ts

```diff
@@ -65,4 +65,18 @@ export class DidDoc {
   public getServicesByClassType<S extends Service = Service>(classType: new (...args: never[]) => S): S[] {
     return this.service.filter((service) => service instanceof classType) as S[]
   }
+
+  /**
+   * Get all DIDComm services ordered by priority descending. This means the highest
+   * priority will be the first entry.
+   */
+  public get didCommServices(): Array<IndyAgentService | DIDCommService> {
+    const didCommServiceClasses = [IndyAgentService.type, DIDCommService.type]
```

Comment:
I would rather name it `didCommServiceTypes` or something like that. When I see classes I expect the class type itself and not its `type` attribute, something like when you call `instanceof SomeClass`.

Reply:
I found `serviceTypes` exported from `src/modules/connections/models/did/service/index.ts` Shouldn't we reuse it? 🤔

Reply:
Ah that's still an old name. It were classes before, but I changed it to types. 

`serviceTypes` is all service types, not just the didcomm ones. I.e. if we add identity hub support it will be a new service type, but not a new didcomm service type